src/containers/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,341 @@
+import React, {FC} from 'react';
+
+import {A, PageTitle} from 'components';
+
+import './PrivacyPolicy.scss';
+
+const PrivacyPolicy: FC = () => {
+  return (
+    <>
+      <PageTitle title="Privacy Policy" />
+      <div className="PrivacyPolicy">
+        <p>Last Updated: April 5, 2021</p>
+        <h1 className="PrivacyPolicy__title">Privacy Policy</h1>
+        <h2 className="PrivacyPolicy__subtitle">Introduction</h2>
+        <p>
+          This privacy policy (“Policy”) describes how thenewboston in California (“Company,” “we,” and “our”)
+          processes, collects, uses and shares personal data when using this website{' '}
+          <A href="https://thenewboston.com" newWindow={false}>
+            https://thenewboston.com
+          </A>{' '}
+          (the “Website”). Please read the following information carefully to understand our practices regarding your
+          personal data and how we will process data.
+        </p>
+        <h2 className="PrivacyPolicy__subtitle">Purpose of Processing</h2>
+        <strong>What is personal data?</strong>
+        <br />
+        We collect information about you in a range of forms, including personal data. As used in this Policy, “personal
+        data” is as defined in the General Data Protection Regulation, this includes any information which, either alone
+        or in combination with other information we process about you, identifies you as an individual, including, for
+        example, your name, postal address, email address and telephone number.
+        <br />
+        <br />
+        <strong>Why do we need your personal data?</strong>
+        <br />
+        We will only process your personal data in accordance with applicable data protection and privacy laws. We need
+        certain personal data in order to provide you with access to the website. If you registered with us, you will
+        have been asked to tick to agree to provide this information in order to access our services, purchase our
+        products, or view our content. This consent provides us with the legal basis we require under applicable law to
+        process your data. You maintain the right to withdraw such consent at any time. If you do not agree to our use
+        of your personal data in line with this Policy, please do not use our website.
+        <h2 className="PrivacyPolicy__subtitle">Collecting Your Personal Data</h2>
+        <p>We collect information about you in the following ways:</p>
+        <strong>Information You Give Us.</strong>
+        <br />
+        This includes:
+        <ul className="PrivacyPolicy__list">
+          <li>
+            the personal data you provide when you register to use our website, including your name, postal address,
+            email address, telephone number, username, password and demographic information (such as your gender)
```

Comment:
and demographic information (such as your gender) - I suggest we omit this if we do not intend to ask for this. This is very sensitive.

Reply:
can we just omit the "gender part" because when we post a job description we ask which region are belonging to